eng/pipelines/internal.yml

```diff
@@ -97,3 +69,36 @@ stages:
       parameters:
         # Symbol validation is not entirely reliable as of yet, so should be turned off until https://github.com/dotnet/arcade/issues/2871 is resolved.
         enableSymbolValidation: false
+
+  - stage: testing
+    jobs:
```

Comment:
We should add a `displayName` property and call it "Test"

Reply:
Ok. It's actually showing up as "Testing" in the job i submitted :)

Reply:
I actually don't see that displayName property being used by the other stages (ex: build). Am I missing something? I can see they're being used on the jobs instead

Reply:
I added the display name, but for some reason it's not showing the proper casing (shows as "test" instead of "Test").